src/core/mesh/qgsmeshlayer.h

```diff
@@ -97,7 +97,15 @@ class CORE_EXPORT QgsMeshLayer : public QgsMapLayer
      */
     struct LayerOptions
     {
-      int unused;  //!< @todo remove me once there are actual members here (breaks SIP <4.19)
+
+      /**
+       * Constructor for LayerOptions.
+       */
+      explicit LayerOptions( const QgsCoordinateTransformContext &transformContext = QgsCoordinateTransformContext() )
```

Comment:
I'd remove the default QgsCoordinateTransformContext() here -- we want to force people to specify a correct one, and not give them any opportunity to just use a default constructed context.